jadx-gui/src/main/java/jadx/gui/ui/codearea/CodePanel.java

```diff
@@ -124,9 +127,9 @@ private void initLineNumbers() {
 	}
 
 	private void initLineNumbers(boolean useSourceLines) {
-		lineNumbers = new LineNumbers(codeArea);
+	/*	lineNumbers = new LineNumbers(codeArea);
 		lineNumbers.setUseSourceLines(useSourceLines);
-		codeScrollPane.setRowHeaderView(lineNumbers);
+		codeScrollPane.setRowHeaderView(lineNumbers);*/
```

Comment:
I'll see what exactly it does, I didn't delete it on purpose because I wasn't sure.

Reply:
I returned everything back, I thought it was a bug), and this is a feature, and fixed it so that it could only be worked on in decompiled java code, xml file not contains source lines.

Reply:
> xml file not contains source lines

Actually, AndroidManifest.xml have source lines sometimes. And there is no point in your new changes, please revert them too.

Reply:
> AndroidManifest.xml have source lines sometimes

As far as I know, I have never seen them there, it is also turned on by default and I personally do not like it. It can even be rendered into a separate option. This function only bothers me.

Reply:
Ok, I see, I can add such option. Just don't disable feature you don't like :smile: